inception-ui-external-search/src/main/java/de/tudarmstadt/ukp/inception/app/ui/externalsearch/SearchPage.java

```diff
@@ -334,6 +336,15 @@ public ResultRowView(String id, long rowNumber, IModel<ExternalSearchResult> mod
             add(new LambdaAjaxLink("importLink", _target -> actionImportDocument(_target, result))
                     .add(visibleWhen(() -> 
                         !documentService.existsSourceDocument(project, documentTitle))));
+            add(new LambdaAjaxLink("openLink", _target -> {
+                PageParameters pageParameters = new PageParameters()
+                    .add(WebAnnoConst.PAGE_PARAM_PROJECT_ID, project.getId())
+                    .add(WebAnnoConst.PAGE_PARAM_DOCUMENT_ID,
+                        documentService.getSourceDocument(project, documentTitle).getId())
+                    .add(WebAnnoConst.PAGE_PARAM_FOCUS, 1);
```

Comment:
Can you add a comment explaining what `PAGE_PARAM_FOCUS` means? It is less obvious than the others.

Reply:
Comment added.
This is the line to focus on when opening in the Annotation page. The annotation page adjust paging so that focused line is centered on the page.

Reply:
@shinyichen : From `AnnotationPage`'s source, it looks like it defaults to the first line anyway:

```
int focus = 0;
if (aFocusParameter != null) {
    focus = aFocusParameter.toInt(0);
}   
```

so does everything work fine if the `PAGE_PARAM_FOCUS` parameter is simply omitted?

Ideally, we'd scroll to wherever within the document the search term was found, but that can wait for a future PR.

Reply:
I removed it.